src/android/CDVBroadcaster.java

```diff
@@ -60,15 +60,17 @@ else if( data instanceof JSONObject ) {
      * @param filter
      */
     protected void registerReceiver(android.content.BroadcastReceiver receiver, android.content.IntentFilter filter) {
-        LocalBroadcastManager.getInstance(super.webView.getContext()).registerReceiver(receiver,filter);
+        // LocalBroadcastManager.getInstance(super.webView.getContext()).registerReceiver(receiver,filter);
+        this.webView.getContext().registerReceiver(receiver, filter);
```

Comment:
does this impact the local events' behavior? If not, I recommend just removing the commented-out lines